internal/manifests/config/tempo-config.yaml

```
@@ -248,7 +248,19 @@ storage:
     s3:
       bucket: {{ .StorageParams.S3.ShortLived.Bucket }}
       endpoint: s3.{{ .StorageParams.S3.ShortLived.Region }}.amazonaws.com
+      insecure: {{ .StorageParams.S3.Insecure }}
+    {{- if .S3StorageTLS.Enabled }}
+    {{- if .S3StorageTLS.CA }}
+      tls_ca_path: {{ .S3StorageTLS.CA }}
+    {{- end }}
+    {{- if .S3StorageTLS.Certificate }}
+      tls_cert_path: {{ .S3StorageTLS.Certificate }}
+    {{- end }}
+    {{- if and .S3StorageTLS.Key }}
+      tls_key_path: {{ .S3StorageTLS.Key }}
+    {{- end }}
     {{- end }}
+  {{- end }}
```

```suggestion
    {{- end }}
```
updated indention to match the opening `{{ if }}`